doc/source/indexing.rst

```diff
@@ -1833,15 +1833,22 @@ that you've done this:
 
 Yikes!
 
+.. _indexing.evaluation_order:
+
 Evaluation order matters
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Furthermore, in chained expressions, the order may determine whether a copy is returned or not.
-If an expression will set values on a copy of a slice, then a ``SettingWithCopy``
-warning will be issued.
-
-You can control the action of a chained assignment via the option ``mode.chained_assignment``,
-which can take the values ``['raise','warn',None]``, where showing a warning is the default.
+When using chained indexing, the order may determine whether a copy is returned
```

Comment:
say 'order of evaluation'

Reply:
I agree that merely 'the order' is not explicit enough. Now that I am looking closer, though, I would say it's not the order of evaluation, but the order in which the user chains the indexing operations (which in turn determines the order the operations are evaluated).

I have turned the sentence into this, which I hope is also in the spirit of your edit:

> When you use chained indexing, the order and type of the indexing operation
partially determine whether the result is a slice into the original object, or
a copy of the slice.

What do you think of that? I don't mean to make this a bikeshedding point, so I will implement whatever you suggest as the final version.